fe/fe-core/src/main/java/org/apache/doris/common/proc/JvmProcDir.java

```diff
@@ -94,6 +94,12 @@ public ProcResult fetchResult() throws AnalysisException {
         Threads threads = jvmStats.getThreads();
         result.addRow(genRow("threads count", threads.getCount()));
         result.addRow(genRow("threads peak count", threads.getPeakCount()));
+        result.addRow(genRow("threads new count", threads.getThreadsNewCount()));
```

Comment:
Please update docs of fe metrics include every new metrics and means of metrics

Reply:
Jvm Thread State is enum type. A thread can be in one of the following states: NEW,RUNNABLE,BLOCKED,WAITING,TIMED_WAITING,TIMED_WAITING,TERMINATED. 
A fe thread can be in only one state at a given point in time. The new metrics mean how many fe threads in a specific state. I think this will be useful when we want to analysis fe whole running status.

Reply:
FE监控项: [http://doris.apache.org/master/zh-CN/administrator-guide/operation/monitor-metrics/fe-metrics.html#%E6%9F%A5%E7%9C%8B%E7%9B%91%E6%8E%A7%E9%A1%B9](url)

Currently, the docs about metrics contains PrometheusMetrics and SystemMetrics.  The doris_fe_snmp belong to SystemMetrics, and the new metrics I added belong to PrometheusMetrics. I want to confirm did I need to add every new metrics in this page?

Reply:
It is ok to add them in this doc.
we will reorganized this doc later.

Reply:
Thank you for your reply. I have update the docs of fe metrics.